src/lib/receptions/index.js

```diff
@@ -12,7 +12,7 @@ const defaultRequestOptions = () => ({
   estimate_of_gb_required: null,
   number_of_smrt_cells: null,
   number_of_flowcells: null,
-  data_type: null,
+  data_type: 'basecalls and raw data',
```

Comment:
I think this is probably fine but it means we will always send data_type, even when we don't want to, because we just [add all request data when building the payload](https://github.com/sanger/traction-ui/blob/develop/src/lib/receptions/sequencescapeUtils.js#L180). This will get filtered out in the service so it probably doesn't matter if its a lot of effort to change.